pkg/config/endpoint_test.go

```diff
@@ -17,3 +17,19 @@ func TestLocationInvalidLocale(t *testing.T) {
 		t.Errorf("have %v, want %v", have, want)
 	}
 }
+
+func TestUnmarshalUserContext(t *testing.T) {
+	var (
+		input = []byte(`{
+			"age": 27,
+			"registered": "2017-12-04T23:11:38Z",
+			"subscription": 2
+			}`)
+		u = userInfo{}
+	)
+
+	err := u.UnmarshalJSON(input)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Maybe we can do a comparison to see if the unmarshaled struct has the fields and values we expect.